.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  ci:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
Is there any cross-platform test needed? GH actions is not always free (2,000 first minutes only)